ulacitbnb/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,234 @@
+                                                              SET Pay_Brand = @Pay_Brand
+                                                                , Pay_Type = @Pay_Type
+                                                            	, Pay_Modality = @Pay_Modality
+                                                                , Pay_Date = @Pay_Date
+                                                                , Pay_Amount = @Pay_Amount
+                                                                , Pay_Taxes = @Pay_Taxes
+                                                                , Pay_Total = @Pay_Total
+                                                            WHERE Pay_ID = @Pay_ID", sqlConnection);
+
+                    sqlCommand.Parameters.AddWithValue("@Pay_ID", payment.Pay_ID);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Brand", payment.Pay_Brand);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Type", payment.Pay_Type);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Modality", payment.Pay_Modality);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Date", payment.Pay_Date);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Amount", payment.Pay_Amount);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Taxes", payment.Pay_Taxes);
+                    sqlCommand.Parameters.AddWithValue("@Pay_Total", payment.Pay_Total);
+
+                    sqlConnection.Open();
+                    int rowsAffected = sqlCommand.ExecuteNonQuery();
+                    sqlConnection.Close();
+                    if (rowsAffected <= 0)
+                    {
+                        return BadRequest($"Payment with ID {payment.Pay_ID} doesn't exist in the database.");
+                    }
+                    else
+                    {
+                        return Ok(rowsAffected);
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                return InternalServerError(ex);
+            }
+        }
+        // ===================================================================================================
+        [HttpDelete, Route("{id}")]
+        public IHttpActionResult Delete(int id)
+        {
+            if (id <= 0)
+            {
+                return BadRequest($"Payment with ID:{id} not found in the database.");
+            }
+
+            try
+            {
+                using (sqlConnection)
+                {
+                    SqlCommand sqlCommand = new SqlCommand(@"DELETE FROM [dbo].[Payment]
+                                                            WHERE Pay_ID = @Pay_ID", sqlConnection);
+
+                    sqlCommand.Parameters.AddWithValue("@Pay_ID", id);
+                    sqlConnection.Open();
+
+                    int rowsAfected = sqlCommand.ExecuteNonQuery();
+                    sqlConnection.Close();
```

Comment:
close connection statemente is unecessary inside `using` it automatically close the connection when the execution cycle go out of the `using`